alamesa/src/components/Restaurant.js

```diff
@@ -38,11 +40,18 @@ function Restaurant ( { name, logo } ) {
       />
       <RestaurantName>
         <h1>{name}</h1>
-        <Button
-          type='button'
-        >
-          Reservar Ahora
-        </Button>
+        <Link to={{
+          pathname: `/restaurants/${id}/reservation`,
+          state: {
+            Restaurant: { id, name, logo},
```

Comment:
Ya este estado no es necesario, el id del restaurante ya va en la url. Idealmente, si necesitamos más información del resturante, la deberiamos pedir al BE

Reply:
Simon. Como aun no tenemos la implementacion de traer info del BE, esto lo podemos configurar luego? y por ahora dejar el `name` y `logo` ya que si los estamos renderizando con la data mockeada.